source/includes/_fonenumbers.md

```diff
@@ -18,7 +18,7 @@ $ curl --request GET
 ```json
 {
   "statusCode": 200,
-  "dids": [
+  "Fonenumbers": [
```

Comment:
I didn't really go through the code samples, figured I'd be checking through them when I added the other languages.